jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -285,7 +285,12 @@ public void invalidatePackageAccessCache() {
 
 	public Class<?> defineClass(ClassLoader classLoader, String className, byte[] classRep, ProtectionDomain protectionDomain, String str) {
 		ClassLoader targetClassLoader = (null == classLoader) ? ClassLoader.bootstrapClassLoader : classLoader;
+/*[IF Java12]
+		// temporary workaround to enable building Java 12
```

Comment:
Is the an open issue to track making the permanent change?

Reply:
I opened https://github.com/eclipse/openj9/issues/3641

Reply:
Will do.